.gitignore

```diff
@@ -20,7 +20,6 @@
 api_data/v1
 doxygen/
 quantum/version.h
-!quantum/tools/eeprom_reset.hex
 *.bin
```

Comment:
```suggestion
!util/reset.eep
*.bin
```

Reply:
theres no point adding an exclude at this point

Reply:
Fine by me.